### DISCUSSION TITLE
Creator Kit availability via Dynamics 365 apps

### DISCUSSION THREAD

**Discussion by gerald560** [microsoft/powercat-creator-kit#210] [General]
Do we know how long once a release has happened to become available in Dynamics 365 apps? (this is our current deployment method) cc @mikefactorial

**Comment by denise-msft** [comment]
@gerald560 can you please explain what you mean as available in Dynamics 365 apps? Are you talking about providing the option to provision this solution when a new D365 instance (aka Power Platform environment) is created?
